upgrade/scripts/upgrade_python_package.py

```diff
@@ -530,6 +580,11 @@ def try_running_module(wheel, *args):
     action="store_true",
     help="Indicates that all packages should be updated",
 )
+parser.add_argument(
+    "--slack-webhook-url",
+    action="store_true",
+    help="Indicates that all failed updates should be sent to slack",
```

Comment:
Very minor - this url is not just a flag that indicates if we should send a notification or not, right?

Reply:
Notifications are currently sent If a user provides `--slack-webhook-url` or if SLACK_WEBHOOK_URL environment variable is set. Do we maybe want another flag for determining if script should send notifications?